frontend/pages/hosts/details/HostDetailsPage/HostDetailsPage.tsx

```diff
@@ -753,6 +753,29 @@ const HostDetailsPage = ({
     name: host?.mdm.macos_setup?.bootstrap_package_name,
   };
 
+  if (!host.mdm.profiles) {
+    host.mdm.profiles = [];
+  } else {
```

Comment:
Please add a TODO to replace the mock data when the backend is ready

Reply:
added.